marc_to_solr/lib/traject_config.rb

```diff
@@ -830,14 +830,20 @@
 end
 
 to_field 'siku_subject_display' do |record, accumulator|
-  genres = process_hierarchy(record, '650|*7|abcvxyz', ['sk'])
+  genres = process_hierarchy(record, '650|*7|abcvxyz') { |field| siku_heading? field }
```

Comment:
Siku is a subject vocabulary, not a genre vocabulary. It's a minor point, and I know it was called 'genres' in the main branch, but I just wanted to point that out.